experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -489,11 +494,101 @@ export interface Sequence<TTypes extends AllowedTypes> extends TreeField {
 
 	readonly length: number;
 
-	// TODO: more and/or better editing APIs. As is, this can't express moves.
-	replaceRange(
+	/**
+	 * Inserts new item(s) at a specified location.
+	 * @param index - The index at which to insert `value`.
+	 * @param value - The content to insert.
+	 */
+	insertAt(
+		index: number,
+		value: FlexibleNodeContent<TTypes> | FlexibleNodeContent<TTypes>[],
+	): void;
+
+	/**
+	 * Inserts new item(s) at the start of the sequence.
+	 * @param value - The content to insert.
+	 */
+	insertAtStart(value: FlexibleNodeContent<TTypes> | FlexibleNodeContent<TTypes>[]): void;
+
+	/**
+	 * Inserts new item(s) at the end of the sequence.
+	 * @param value - The content to insert.
+	 */
+	insertAtEnd(value: FlexibleNodeContent<TTypes> | FlexibleNodeContent<TTypes>[]): void;
+
+	/**
+	 * Removes the item at the specified location.
+	 * @param index - The index at which to remove the item.
+	 */
+	removeAt(index: number): void;
+
+	/**
+	 * Removes all items between the specified indices.
+	 * @param start - The starting index of the range to remove (inclusive). Defaults to the start of the sequence.
+	 * @param end - The ending index of the range to remove (exclusive). Defaults to the index of the last item.
+	 * Omitting `start` and `end` will result in removal of all items in the sequence.
+	 */
+	removeRange(start?: number, end?: number): void;
+
+	/**
+	 * Moves the specified items to the start of the sequence.
+	 * @param sourceStartIndex - The starting index of the range to move (inclusive).
+	 * @param sourceEndIndex - The ending index of the range to move (exclusive)
+	 */
+	moveToStart(sourceStartIndex: number, sourceEndIndex: number): void;
```

Comment:
Nit: Recommend just 'sourceStart' and 'sourceEnd'.  (JS devs typically see brevity as a sign that something is thoughtfully designed.)

Reply:
Hehe, just pulling from the spec directly. I'll make the change.